tamr_client/__init__.py

```diff
@@ -37,8 +37,8 @@
 import tamr_client.session as session
 
 # datasets
-from tamr_client.datasets.dataset import Dataset, DatasetNotFound
-import tamr_client.datasets.dataset as dataset
+from tamr_client.datasets import Dataset
```

Comment:
Let's also rename the package from `tc.datasets` to just `tc.dataset`.

Reply:
so would it just be `import tamr_client.dataset` or `import tamr_client.dataset as dataset`?

Reply:
We want users to be able to do:

```python
import tamr_client as tc # single import for the whole library

x = tc.dataset.<some dataset function>(...)
```

`import tamr_client as tc` will make everything that is available in `tamr_client/__init__.py` available *by the same name* it has within `__init__.py`. 

So if you just do `import tamr_client.dataset` within `tamr_client/__init__.py`, then the name `dataset` is *not* available within `tamr_client/__init__.py` (you would still have to reference as `tamr_client.dataset` within `tamr_client/__init__.py`).

That means that the user would have to do:

```python
import tamr_client as tc # single import for the whole library

x = tc.tamr_client.dataset.<some dataset function>(...)
```

So you need to do `import tamr_client.dataset as dataset` so that the name `dataset` is available in `tamr_client/__init__.py` so that you can do `tc.dataset.<some dataset function>(...)`.